README.md

```diff
@@ -1,2 +1,64 @@
-# First-capstone-Project
-This project shows my acquired knowledge so far on Html, Css and Javascript
+# First-capstone-Project: Milestone 6
+
+This project shows my acquired knowledge so far;
+
+> The capstone project demonstrates Javascript, Html and Css.
+> This is also a project that shows my proficiency as I go through Microverse. It is a work-in-progress and I hope to make it a masterpiece.
+
+## Live Demo
+
+[Link to Live Demo](https://eternalgratis.github.io/First-capstone-Project/)
+
+## Built With
+
+- HTML
+- CSS
+- Linter
+- Javascript
+
```

Comment:
- [ ] For the project documentation you had to record a video please recored it and respect all the requirements 👇 

![image](https://user-images.githubusercontent.com/75126481/177012688-f347acf4-2f17-440c-9011-e05cc26cbcba.png)